ere-guests/Cargo.toml

```diff
@@ -161,4 +162,8 @@ ethereum_ssz = "0.9"
 
 risc0-ethereum-trie = { git = "https://github.com/risc0/risc0-ethereum.git", rev = "4a5efe13b3655258b89e7082c57743e445133dc8" }
 
+# Patch `ziskos` of `zisk-patch-*` to pin to the same version of guest.
+[patch."https://github.com/0xPolygonHermez/zisk.git"]
+ziskos = { git = "https://github.com/0xPolygonHermez//zisk.git", tag = "v0.11.0" }
```

Comment:
Patch `ziskos` to pin the patches' dependency version to the same one of the guest ([the patches don't specify any version](https://github.com/0xPolygonHermez/zisk-patch-hashes/blob/zisk-patch-sha2/v0.10.9/sha2/Cargo.toml#L26) and cargo will recognize them as different crates).

Reply:
Uhm, never saw a patch that changes a source to the same one.

Say that a zisk patch ends up resolving to v0.12.0 in its cargo lock -- I don't think this patch will force them to use ziskos v0.11.0, no?

The way I read patches is usually to replace one source with another for a specific semver. I don't think they can pin versions of dependencies, only replace from where they're resolved if are part of the dependency graph.

(I'm not an expert of cargo patches, so sharing my thinking here to double check)

Nit: double `/` in url.